src/store/traction/pacbio/runCreate/run.js

```diff
@@ -44,9 +44,166 @@ const defaultWellAttributes = () => {
     ccs_analysis_output_include_low_quality_reads: 'No',
     demultiplex_barcodes: onInstrument,
     include_fivemc_calls_in_cpg_motifs: 'Yes',
+    movie_acquisition_time: null,
+    include_base_kinetics: null,
+    library_concentration: null,
+    polymerase_kit: null,
   }
 }
 
+const smrtLinkVersionDefaultComponents = {
```

Comment:
Feel like it might be more sensible to have this data in the component its used it instead of here.